jsk_data/node_scripts/data_collection_server.py

```diff
@@ -277,10 +296,14 @@ def _save(self):
 
     def start_service_cb(self, req):
         self.start = True
+        if self.rosbag:
+            self.start_rosbag()
         return TriggerResponse(success=True)
 
     def end_service_cb(self, req):
         self.start = False
+        if self.rosbag:
```

Comment:
simpler?
```
        if self.rosbag_topics:
```

Reply:
Thank you. is that so.
I will remove the param of rosbag and change it to take rosbag if rosbag_topics is specified.